src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -901,6 +901,9 @@ private void testResponseRate() {
 
         // Change the feedback path of the question and save
         feedbackEditPage = getFeedbackEditPage();
+        // Alert class should be enabled to warn instructors against changing visibility options after collecting responses.
+        // Passing 1 as question number as only one question is created.
```

Comment:
On second thoughts, instead of leaving a comment, moving this into its own test section would be better:
`______TS("check warning against changing visibility options for question with existing response");`